tutor/src/components/tours/custom/centered-wheel.jsx

```diff
@@ -0,0 +1,46 @@
+import React        from 'react';
+import ReactDOM     from 'react-dom';
+import { Tooltip }  from 'react-joyride';
+
+import kebabCase    from 'lodash/kebabCase';
+import first        from 'lodash/first';
+import classnames   from 'classnames';
+
+export default class CenteredWheel extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.className = kebabCase(this.constructor.name);
+    this.wrapperClassName = this.props.className? `${first(this.props.className.split(' '))}-wrapper` : 'training-wheel-wrapper';
+  }
+
```

Comment:
I'm not seeing why className needs to be based off `this.constructor.name`.  That seems fragile since the constructor name may be changed when the code's uglified for production.  Why not just make `className` be a property, i.e.:

```javascript
export default class CenteredWheel extends React.PureComponent {
  className = 'centeredWheel'

  setupWrapperClasses() {

...
```



Rather than making `wrapperClassName` static, you could make a computed getter property; that way it'll update when/if props className changes:

```javascript
@computed get wrapperClassName() {
  return this.props.className? `${first(this.props.className.split(' '))}-wrapper` : 'training-wheel-wrapper';
}
```